### ISSUE TITLE
fanc aggregate

### ISSUE THREAD

**Issue by knight-hzj** [vaquerizaslab/fanc#146]
Hi, 

FAN-C is a great tool for HiC data analysis. I am trying to use it to analyzed my data. Here, I got an issue when I did fanc aggregate on my .hic file( from Juicer). BTW,  fanc compartment, fanc plot, fanc insulation are working well on this .hic file,  so, I think the input .hic file should be OK.
So, May I kindly ask how to figure this out? Much appreciated.

The way that I am doing fanc aggregate is as follows:
fanc aggregate WT_.hic@100kb \
               domains/WT_tads.bed \
               aggregate/WT_100kb.agg \
               -p aggregate/WT_100kb.agg.pdf \
               -m aggregate/WT_100kb.agg.txt \
                -e -l

the error information is below,
2023-03-14 02:48:44,135 INFO FAN-C version: 0.9.25
2023-03-14 02:48:44,241 INFO Creating aggregate matrix from differently-sized regions
2023-03-14 02:48:44,504 INFO Checking region pair validity...
2023-03-14 02:48:44,508 INFO 10892/10892 region pairs are invalid
/python3.9/site-packages/fanc/compatibility/juicer.py:686: RuntimeWarning: divide by zero encountered in divide
  return np.array(vectors[resolution]) / sf
Matrices 100% (0 of N/A) |#              | Elapsed Time: 0:00:00 ETA:  --:--:--
Matrices 100% (0 of N/A) |#              | Elapsed Time: 0:00:00 ETA:  --:--:--
/python3.9/site-packages/fanc/architecture/aggregate.py:369: RuntimeWarning: invalid value encountered in divide
  am = matrix_sum/counter_matrix
/python3.9/site-packages/fanc/plotting/statistics.py:363: RuntimeWarning: All-NaN slice encountered
  vmin = np.nanmin(m)
/python3.9/site-packages/fanc/plotting/statistics.py:365: RuntimeWarning: All-NaN slice encountered
  vmax = np.nanmax(m).

The WT_100kb.agg.txt is like this:
nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan
nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan
nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan
nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan
nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan
nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan nan

**Comment by knight-hzj** [comment]
Hi,
Thanks so much for your quick response. Please see below, 
chr10	132020000	133670000	.	.
chr10	36180000	37080000	.	.
chr10	62690000	64000000	.	.
chr10	124770000	125730000	.	.
chr10	23440000	24340000	.	.
chr10	6720000	7520000	.	.
chr10	86120000	86290000	.	.
chr10	29790000	30640000	.	.
chr10	111280000	111500000	.	.
chr10	1750000	3080000	.	.
chr10	71270000	71720000	.	.
chr10	28530000	28680000	.	.
chr10	1780000	2950000	.	.
chr10	102830000	103550000	.	.
chr10	70730000	70930000	.	.
chr10	129670000	129850000	.	.
chr10	54560000	59380000	.	.
chr10	76870000	76990000	.	.
chr10	91890000	92490000	.	.
chr10	85900000	86310000	.	.

**Comment by kaukrise** [comment]
Juicer files typically remove the `chr` prefix - maybe that is causing your issue. can you try with

```
10 132020000 133670000 . .
10 36180000 37080000 . .
10 62690000 64000000 . .
10 124770000 125730000 . .
10 23440000 24340000 . .
10 6720000 7520000 . .
10 86120000 86290000 . .
10 29790000 30640000 . .
10 111280000 111500000 . .
10 1750000 3080000 . .
10 71270000 71720000 . .
10 28530000 28680000 . .
10 1780000 2950000 . .
10 102830000 103550000 . .
10 70730000 70930000 . .
10 129670000 129850000 . .
10 54560000 59380000 . .
10 76870000 76990000 . .
10 91890000 92490000 . .
10 85900000 86310000 . .
```

**Comment by knight-hzj** [comment]
Yes!  You are right, it is working well after I removed them. Thank you so so so so much!!!!!!!!
Thank you again for your such quickly answer!
Best!
